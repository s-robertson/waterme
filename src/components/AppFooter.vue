<template>
  <div class="app-footer">
    <div class="app-footer__content">
      <div class="app-footer__oss-attributes">
        Icons made by
        <a href="https://www.flaticon.com/authors/freepik" title="Freepik"
          >Freepik</a
        >
        from
        <a href="https://www.flaticon.com/" title="Flaticon">
          www.flaticon.com</a
        >
      </div>
    </div>
  </div>
</template>

<script>
export default {
  name: "AppFooter"
};
</script>

<style lang="scss">
@import "../assets/variables";

.app-footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 2rem;
}

.app-footer__content {
  margin: 0 auto;
  max-width: $app-max-width;
}

.app-footer__oss-attributes {
  font-size: 0.8rem;
}
</style>
