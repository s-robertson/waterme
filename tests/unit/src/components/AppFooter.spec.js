import { shallowMount } from "@vue/test-utils";
import AppFooter from "@/components/AppFooter";

describe("AppFooter component", () => {
  it("Should render correctly", () => {
    const wrapper = shallowMount(AppFooter);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
