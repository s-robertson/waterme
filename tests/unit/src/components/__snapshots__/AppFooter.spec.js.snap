// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter component Should render correctly 1`] = `
<div class="c-app-footer">
  <div class="c-app-footer__content">
    <div class="c-app-footer__oss-attributes">
      Icons made by
      <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a>
      from
      <a href="https://www.flaticon.com/" title="Flaticon">
        www.flaticon.com</a></div>
  </div>
</div>
`;
